specs/HIVE-267-upgrade-swap/proposal.md

```
@@ -0,0 +1,120 @@
+type: spec
+status: draft # draft | implementing | verifying | archived
+created: "2026-06-24"
+issue: "hive#267"   # repo#NNN — GitHub issue / Project item that tracks this spec
+tags: [spec, proposal]
+template_version: "1.0"
+---
+
+# HIVE-267: Upgrade swap
+
+> **Naming**: file lives at `<repo>/specs/HIVE-267-upgrade-swap/proposal.md`.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)